libs/@guardian/atoms-rendering/src/YoutubeAtomOverlay.tsx

```diff
@@ -81,17 +84,46 @@ const playButtonStyling = css`
 	}
 `;
 
-const videoDurationStyles = css`
+const pillStyles = css`
 	position: absolute;
 	top: ${space[2]}px;
 	right: ${space[2]}px;
 	${textSans.xxsmall({ fontWeight: 'bold' })};
 	background-color: rgba(0, 0, 0, 0.7);
 	color: ${palette.neutral[100]};
-	padding: ${space[1]}px ${space[3]}px;
 	border-radius: ${space[3]}px;
+	padding: 0 6px;
+	display: inline-flex;
 `;
 
+const pillItemStyles = css`
+	:last-child:not(:only-of-type) {
+		border-left: 1px solid rgba(255, 255, 255, 0.5);
+	}
+`;
+
+const pillTextStyles = (isLive: Boolean = false) => css`
+	line-height: ${space[4]}px;
+	padding: ${space[1]}px 6px;
+
+	::before {
+		${isLive === true &&
+		`
+			content: '';
+			width: 9px;
+			height: 9px;
+			border-radius: 50%;
+			background-color: ${palette.news[500]};
+			display: inline-block;
+			position: relative;
+			margin-right: 0.1875rem;
+		`}
+	}
+`;
```

Comment:
The recommended way to do emotion styling is [composing styles](https://emotion.sh/docs/@emotion/css#cx), rather than methods that take arguments.